pkg/transfer/flex/main.go

```diff
@@ -38,6 +41,30 @@ const (
 	StatusNotSupported = "Not supported"
 )
 
+//File systems to back file system in a file
+type FileSystem string
+
+const (
+	//Standard, supported everywhere
+	FileSystemExt4 FileSystem = "ext4"
+)
+
+//Amount of space available for writing data
```

Comment:
In go, "comments should begin with the name of the thing being described and end in a period". You can find more information [here](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences). Same as above, you have a bunch of tools (golint, go fmt, go vet...) that can help you to get a clean code without having to think about it 🙂

Reply:
Fixed